packages/components/divider/src/Divider.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <div class="ix-divider" :class="className">
+    <span v-if="$slots.default" class="ix-divider-inner-text" :style="style">
+      <slot />
+    </span>
+  </div>
+</template>
+<script lang="ts">
+import type { PropType } from 'vue'
+import type { DividerConfig } from '@idux/components/core/config'
+import type { DividerPosition, DividerType } from '@idux/components/core/types'
+import type { DividerProps } from './types'
+
+import { computed, defineComponent } from 'vue'
+import { useGlobalConfig } from '@idux/components/core/config'
+
+const staticPosition = new Set<string>(['left', 'center', 'right'])
+
+export default defineComponent({
+  name: 'IxDivider',
+  props: {
+    dashed: Boolean,
+    plain: Boolean,
+    position: {
+      type: String as PropType<DividerPosition>,
+      required: true,
+    },
+    type: {
+      type: String as PropType<DividerType>,
+      required: true,
+    },
```

Comment:
`position` 和 `type` 都不是必传的, 因为我们有提供全局配置的默认值，这里只要给一个 `default: undefined` 即可
```suggestion
    position: {
      type: String as PropType<DividerPosition>,
     default: undefined,
    },
    type: {
      type: String as PropType<DividerType>,
      default: undefined,
    },
```

Reply:
这里是我不太了解Vue3对ts要怎么做兼容，我本来的想法就是所有都是非必填的